### ISSUE TITLE
Replace the kill button with close

### ISSUE THREAD

**Issue by ghost** [PrismLauncher/PrismLauncher#860]
### Role

I use the launcher.

### Suggestion

I want the "kill" button to be replaced with "close", which simply acts as an Alt+F4 on the game, with a dropdown option to kill.

### Benefit

I occasionally find myself using the kill button as a close button, and such a change would make it safer to do so.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by JohnShiozo** [comment]
![image](https://user-images.githubusercontent.com/66097391/218605770-5dd69b5b-fc99-4ca9-8c85-bc95909d0cf7.png)

**Comment by ghost** [comment]
> ![image](https://user-images.githubusercontent.com/66097391/218605770-5dd69b5b-fc99-4ca9-8c85-bc95909d0cf7.png)

nice meme

**Comment by Xetaxheb** [comment]
why would you ever be using the kill button as a close button? why are you using a close button at all instead of quitting out properly?
i'll grant you that these days basically everything saves properly (changed keybinds in <1.13 i'm looking at you) and unless you have disabled the game pausing (f3+p or gui open) you're probably not gonna lose anything... but why are you X'ing out instead of closing properly? let alone going clear into the kill prompt instead of using taskbar or the minecraft window's X? (or alt+f4)

regardless, kill serves a purpose when the game wouldn't respond to alt+f4 anyway and burying it behind a dropdown is inconvenient enough that the obscure benefit from adding close isn't worth it

**Comment by ghost** [comment]
Going into the instance and actually getting to the quit game button takes a few extra seconds and the convenience would be worth it.

**Comment by Xetaxheb** [comment]
~~regardless your premise is wrong as there's not a functional difference here anyway in java besides making it potentially not responsive during a frozen instance.
**killing javaw.exe isn't somehow less safe to your data than alt+f4ing it.** minecraft doesn't do special handling when you x out. it's equivalent but just running through less reliable api calls.~~ edit: this is wrong

**Comment by TheKodeToad** [comment]
Minecraft does handle the close action - the game doesn't just call System.exit the moment glfw reports the window being closed. It saves the level and also sends quitting packets. However, pressing kill in Prism Launcher does not allow this to happen.

**Comment by Xetaxheb** [comment]
You're right, and I'm not sure why I thought it didn't... maybe an old version didn't, I'm not sure.

Regardless using the console window to close it shouldn't be any faster than right clicking taskbar and closing, using the x on the window, going back in and altf4ing, task manager end tasking (not process), etc. This is the main convenient way to kill it when it's hung or in a state that altf4 wouldn't close it anyway (some loading processes especially)

**Comment by TheKodeToad** [comment]
hmm... there's also how much work it would take to implement. although if this was figured out it could also help with auto-maximisation.

**Comment by ghost** [comment]
I still find myself using the kill button to close my instances, it is more effort to go there and quit properly. I'm doing some active work with the instance and i need a quick way to close it when i can't do something due to it running, which the kill button provides lmao

**Comment by Xetaxheb** [comment]
you could make an ahk script with a hotkey

**Comment by ghost** [comment]
i have no idea what you're talking about and i probably wouldn't bother if i did

**Comment by RokeJulianLockhart** [comment]
Regardless, I think we can all agree that we can't *replace* the button, because they don't serve the same purpose. The issue should be updated to reflect that.

**Comment by Xetaxheb** [comment]
not really, the issue should be closed because there's no point 😂
